plugins/auth-backend/src/providers/auth0/provider.ts

```diff
@@ -74,6 +95,7 @@ export class Auth0AuthProvider implements OAuthHandlers {
         // We need passReqToCallback set to false to get params, but there's
         // no matching type signature for that, so instead behold this beauty
         passReqToCallback: false as true,
+        store: this.store,
```

Comment:
Hm. @Rugvip does this make sense?

Reply:
this copys the implementation on the okta one. and without it it kept blowing up because of the state

Reply:
Yep seems like a needed workaround